src/components/TodoItem.tsx/TodoItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onSelect: (todo: Todo) => void;
+  selectedTodo: Todo | null;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo, onSelect, selectedTodo }) => {
+  return (
+    <tr data-cy="todo" className="" key={todo.id}>
```

Comment:
Avoid using the `key` attribute directly on the `tr` element during render. Instead, pass the `key` prop to the `TodoItem` component from the parent component, such as `TodoList`, to ensure consistency and avoid potential issues with React's reconciliation process.